cmake/Catch.cmake

```diff
@@ -109,43 +106,45 @@ function(catch_discover_tests TARGET)
 
   # Generate a unique name based on the extra arguments
   string(SHA1 args_hash "${_TEST_SPEC} ${_EXTRA_ARGS}")
-  string(SUBSTRING ${args_hash}
-                   0
-                   7
-                   args_hash)
+  string(SUBSTRING ${args_hash} 0 7 args_hash)
 
   # Define rule to generate test list for aforementioned test executable
   set(ctest_include_file "${CMAKE_CURRENT_BINARY_DIR}/${TARGET}_include-${args_hash}.cmake")
   set(ctest_tests_file "${CMAKE_CURRENT_BINARY_DIR}/${TARGET}_tests-${args_hash}.cmake")
-  get_property(crosscompiling_emulator TARGET ${TARGET} PROPERTY CROSSCOMPILING_EMULATOR)
-  add_custom_command(TARGET
-                     ${TARGET}
-                     POST_BUILD
-                     BYPRODUCTS
-                     "${ctest_tests_file}"
-                     COMMAND "${CMAKE_COMMAND}" -D "TEST_TARGET=${TARGET}" -D
-                             "TEST_EXECUTABLE=$<TARGET_FILE:${TARGET}>" -D
-                             "TEST_EXECUTOR=${crosscompiling_emulator}" -D
-                             "TEST_WORKING_DIR=${_WORKING_DIRECTORY}" -D "TEST_SPEC=${_TEST_SPEC}"
-                             -D "TEST_EXTRA_ARGS=${_EXTRA_ARGS}" -D "TEST_PROPERTIES=${_PROPERTIES}"
-                             -D "TEST_PREFIX=${_TEST_PREFIX}" -D "TEST_SUFFIX=${_TEST_SUFFIX}" -D
-                             "TEST_LIST=${_TEST_LIST}" -D "CTEST_FILE=${ctest_tests_file}" -P
-                             "${_CATCH_DISCOVER_TESTS_SCRIPT}"
-                     VERBATIM)
-
-  file(WRITE "${ctest_include_file}"
-             "if(EXISTS \"${ctest_tests_file}\")\n"
-             "  include(\"${ctest_tests_file}\")\n"
-             "else()\n"
-             "  add_test(${TARGET}_NOT_BUILT-${args_hash} ${TARGET}_NOT_BUILT-${args_hash})\n"
-             "endif()\n")
+  get_property(
+    crosscompiling_emulator
+    TARGET ${TARGET}
+    PROPERTY CROSSCOMPILING_EMULATOR)
+  add_custom_command(
+    TARGET ${TARGET} POST_BUILD
+    BYPRODUCTS "${ctest_tests_file}"
+    COMMAND
+      "${CMAKE_COMMAND}" -D "TEST_TARGET=${TARGET}" -D "TEST_EXECUTABLE=$<TARGET_FILE:${TARGET}>" -D
+      "TEST_EXECUTOR=${crosscompiling_emulator}" -D "TEST_WORKING_DIR=${_WORKING_DIRECTORY}" -D
+      "TEST_SPEC=${_TEST_SPEC}" -D "TEST_EXTRA_ARGS=${_EXTRA_ARGS}" -D
+      "TEST_PROPERTIES=${_PROPERTIES}" -D "TEST_PREFIX=${_TEST_PREFIX}" -D
+      "TEST_SUFFIX=${_TEST_SUFFIX}" -D "TEST_LIST=${_TEST_LIST}" -D "CTEST_FILE=${ctest_tests_file}"
+      -P "${_CATCH_DISCOVER_TESTS_SCRIPT}"
+    VERBATIM)
+
+  file(
+    WRITE "${ctest_include_file}"
+    "if(EXISTS \"${ctest_tests_file}\")\n" "  include(\"${ctest_tests_file}\")\n" "else()\n"
+    "  add_test(${TARGET}_NOT_BUILT-${args_hash} ${TARGET}_NOT_BUILT-${args_hash})\n" "endif()\n")
```

Comment:
If you compare left and right, you will agree that this `6.0` version is worse! But anyway!

Reply:
version `0.6` you meant. I agree.. but anyway...

Reply:
version `0.6` you meant... I agree, but anyway...